solutions/usaco-966.mdx

```diff
@@ -47,4 +47,48 @@ ans += first8[N % 8]
 print(ans)
 ```
 </PySection>
+<JavaSection>
+  
+```java
+
+import java.util.*;
+import java.io.*;
+
+
+class usaco-966
+{
+	public static void main (String[] args) throws IOException
+	{
+	
+		BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
+		StringTokenizer st = new StringTokenizer(br.readLine());
+		int N = Integer.parseInt(st.nextToken());
+		
+		Vector<Integer> v = new Vector<Integer>(N);
```

Comment:
You can't initialize your list with `N` as capacity. N can up to 10^9 which will lead to RE/memory limit exceeded. And you only want to save a few **fixed** numbers anyway. (Actually, *why not use an array?*)